graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -17,10 +17,11 @@
 
 import pytest
 import pandas as pd
-from numpy.testing import assert_array_equal, assert_array_almost_equal
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal, assert_almost_equal
 from pyspark.sql import SparkSession, DataFrame, functions as F
 from pyspark.sql.types import ArrayType, FloatType, StructField, StructType, StringType
-
+from scipy.special import erfinv
```

Comment:
```suggestion
from scipy.special import erfinv

```

Add a blank line between third-party and local/library imports: https://peps.python.org/pep-0008/#imports